server/medical-contact/router.ts

```diff
@@ -0,0 +1,136 @@
+import type {NextFunction, Request, Response} from 'express';
+import express from 'express';
+import MedicalContactCollection from './collection';
+import * as accountValidator from '../account/middleware';
+import * as medicalContactValidator from '../medical-contact/middleware';
+import * as util from './util';
+import { MedicalContact } from './model';
+
+const router = express.Router();
+
+/**
+ * Get all of an account's medical contacts.
+ *
+ * @name GET /api/medical-contacts
+ *
+ * @return {MedicalContactResponse[]} - An array of medical contacts created by the logged in user
+ * @throws {401} - If the account is not logged in
+ *
+ */
+router.get(
+    '/',
+    [
+        accountValidator.isAccountLoggedIn
+    ],
+    async (req: Request, res: Response) => {
+        const medicalContacts = await MedicalContactCollection.findAllByOwnerId(req.session.accountId as string);
+        const response = medicalContacts.map(util.constructMedicalContactResponse);
+        res.status(200).json(response);
+    }
+);
+
+/**
+ * Create a new medical contact.
+ *
+ * @name POST /api/medical-contacts
+ * @param {string} ownerId - The id of the owner of the medical contact
+ * @param {string} title - The username of the user
+ * @param {string} first_name - The username of the user
+ * @param {string} last_name - The username of the user
+ * @param {string} hospital - The username of the user
+ * @param {string} specialty - The username of the user
+ * @param {string} phone_number - The username of the user
+ * @param {string} notes - The username of the user
+ * @return {MedicalContactResponse} - The created medical contact
+ * @throws {401} - If the account is not logged in
+ * @throws {400} - If the first name of the medical contact is empty or a stream of empty spaces
+ * @throws {400} - If the last name of the medical contact is empty or a stream of empty spaces
+ * @throws {400} - If the phone number of the medical contact is not a valid phone number
```

Comment:
could you please also update the README to reflect the additional validity constraints you added for phone numbers and names